src/instance.ts

```diff
@@ -193,6 +248,255 @@ class Instance extends common.GrpcServiceObject {
+        if (rowBackups) {
+          backups = rowBackups.map(rowBackup => {
+            return this.backup(rowBackup.name!);
+          });
+        }
+
+        callback(err, backups, ...args);
+      }
+    );
+  }
+
+  getBackupOperations(
+    options?: GetBackupOperationsOptions
+  ): Promise<GetBackupOperationsResponse>;
+  getBackupOperations(callback: GetBackupOperationsCallback): void;
+  getBackupOperations(
+    options: GetBackupOperationsOptions,
+    callback: GetBackupOperationsCallback
+  ): void;
+  /**
+   * Query object for listing backup operations.
+   *
+   * @typedef {object} GetBackupOperationsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetBackupOperationsResponse
+   * @property {IOperation[]} 0 Array of {@link IOperation} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List pending and completed backup operations for all databases in the instance.
+   *
+   * @see {@link #listOperations}
+   *
+   * @param {GetBackupOperationsOptions} [options] Contains query object for
+   *     listing backup operations and request configuration options, outlined
+   *     here: https://googleapis.github.io/gax-nodejs/CallSettings.html.
+   * @returns {Promise<GetBackupOperationsResponse>} when resolved, contains a
+   *     paged list of backup operations.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const [operations] = await instance.getBackupOperations();
+   */
+  getBackupOperations(
+    optionsOrCallback?:
+      | GetBackupOperationsOptions
+      | GetBackupOperationsCallback,
+    cb?: GetBackupOperationsCallback
+  ): void | Promise<GetBackupOperationsResponse> {
+    const callback =
+      typeof optionsOrCallback === 'function'
+        ? (optionsOrCallback as GetBackupOperationsCallback)
+        : cb!;
+    const options =
+      typeof optionsOrCallback === 'object'
+        ? (optionsOrCallback as GetBackupOperationsOptions)
+        : {gaxOptions: {}};
+    const gaxOpts: CallOptions = options.gaxOptions as CallOptions;
+    const reqOpts = extend({}, options, {
+      parent: this.formattedName_,
+    });
+    delete reqOpts.gaxOptions;
+
+    this.request<IOperation[]>(
+      {
+        client: 'DatabaseAdminClient',
+        method: 'listBackupOperations',
+        reqOpts,
+        gaxOpts,
+      },
+      (err, operations, ...args) => {
+        callback!(err, operations, ...args);
+      }
+    );
```

Comment:
@alexander-fenster, same here. `getBackupOperations()` is setup identically to `getDatabaseOperations()` below. But instead of returning an `IOperation[]`, the type of `operations` in the callback is `IListBackupOperationsResponse`. `getDatabaseOperations()` returns an `IOperation[]` correctly.